packages/api-worker/src/cloudflare-index.ts

```diff
@@ -0,0 +1,3 @@
+import app from './server';
+
+export default app;
```

Comment:
Any Cloudflare specific here? I was expecting that we would need initialize connection to D1 dtabase or have same interface for it?